index.html

```diff
@@ -202,7 +149,7 @@ <h3 class="title ">Project name goes here...</h3>
                     <div class="rails language-title ">Ruby/Rails</div>
                     <div class="js language-title ">Javascript</div>
                 </div>
-                <a class="see-this-project " href="# ">
+                <a class="see-this-project" id="see-this-project-1" href="#">
                     <p class="text ">see this projects</p>
                     <img class="arrow-d " src="images/Union.png " alt="icon-arrow ">
 
```

Comment:
- [ ] Delete this code pertaining to the projects from the HTML file. Everything related to the project should be populated from the array of objects within the javascript file.